test/boost/index_with_paging_test.cc

```diff
@@ -20,9 +20,16 @@ SEASTAR_TEST_CASE(test_index_with_paging) {
 
         sstring big_string(4096, 'j');
         // There should be enough rows to use multiple pages
-        for (int i = 0; i < 64 * 1024; ++i) {
-            e.execute_cql(format("INSERT INTO tab (pk, ck, v, v2, v3) VALUES ({}, 'hello{}', 1, {}, '{}')", i % 3, i, i, big_string)).get();
-        }
+        auto prepared_id = e.prepare("INSERT INTO tab (pk, ck, v, v2, v3) VALUES (?, ?, 1, ?, ?)").get0();
+        auto big_string_v = cql3::raw_value::make_value(serialized(big_string));
+        parallel_for_each(boost::irange(0, 64 * 1024), [&] (auto i) {
+            return e.execute_prepared(prepared_id, {
```

Comment:
parallel_for_each with a concurrency of 65k is excessive. Likely max_concurrent_for_each will yield better results.

Do we really need 65k entries? If we change the text field to be much larger, perhaps we can get away with a smaller count. Or we can configure a smaller page size.

Reply:
Addressed in https://github.com/scylladb/scylladb/commit/ca206995db52fc9c19bed34215b9261b46251103